File: lista_4/python/totient_lib/main.py
import sys
from totient import totient

def main():
    argc = len(sys.argv)
    argv = sys.argv

    if argc < 2:
        print("\nWrong number of arguments.\nUsage: /program <number>", end="")

    print("\n", end="")
    for i in range(1, argc):
        print(f"totient({argv[i]}) = {totient(int(argv[i]))};")

    print("\n", end="")

if __name__ == "__main__":
    main()

File: lista_4/c/totient_lib/main.c
//
// Created by serhi on 12/9/2025.
//

#include <stdio.h>
#include <stdlib.h>

#include "totient.h"

int main(int argc, char *argv[]) {
    printf("\n");
    for (int i = 1; i < argc; i++) {
        printf("totient(%s) = %d;\n",argv[i], totient(atoi(argv[i])));
    }
    printf("\n");
    return 0;
}